testsuite/classlibrary/TestUGen_initSample.sc

```diff
@@ -0,0 +1,253 @@
+TestUGen_initSample : UnitTest {
+
+	var server;
+
+	*initClass {
+		passVerbosity = UnitTest.brief;
+	}
+
+	setUp {
+		server = Server(this.class.name);
+		server.bootSync;
+	}
+
+	tearDown {
+		server.quit;
+		server.remove;
+	}
+
+	// Async helpers
```

Comment:
Are these async helpers useful enough to promote to a superclass that all UGen unit tests might have access to?